.github/workflows/release.yml

```diff
@@ -380,3 +380,80 @@ jobs:
           repository: fermyon/homebrew-tap
           event-type: spin-release
           client-payload: '{"version": "${{ github.ref_name }}"}'
+
+  docker:
+    runs-on: "ubuntu-20.04"
```

Comment:
possibly use 22.04?

Reply:
I propose relegating to a follow-up; this is the version used elsewhere in the workflow so it might be nice to update all at the same time.

Reply:
Yes 👍🏻 that makes sense. Thank you